trace-server/org.eclipse.tracecompass.incubator.trace.server.jersey.rest.core/src/org/eclipse/tracecompass/incubator/internal/trace/server/jersey/rest/core/services/DataProviderService.java

```diff
@@ -1326,11 +1326,11 @@ public Response createDataProvider(
     @Tag(name = OCG)
```

Comment:
At line 1314, replace 'from created by' with 'created from'?
Also check the param and return tags.

Reply:
Done